### DISCUSSION TITLE
Out of range float values are not JSON compliant

### DISCUSSION THREAD

**Discussion by mdavid800** [developmentseed/titiler#501] [Q&A] [answered]
HI Guys, 

Wondering if you can help steer me in the right direction. 

I am using a COG file hosted on AWS alongside TiTiler running on Lambda .However , when I try and extract the Minv & MaxV for rescaling (my data is in Float 32)  I get the following error : 

`{'detail': 'Out of range float values are not JSON compliant'}`

I can get the Minv & Maxv values using RioTiler using the same url fine but can not get them using TiTiler . Any ideas where I am going wrong ?  

my notebook can be found here: 
https://github.com/mdavid800/Dash_Riotiler_Titiler/blob/2e780bc723a89db6b4349a569230af7b8c9d5199/DashLeaflet_TiTiler.ipynb   

and screenshot of code below : 
![image](https://user-images.githubusercontent.com/88830934/181485473-acf4ec76-f211-4854-b77a-f694eabff958.png)

Thanks , 

David

**Comment by vincentsarago** [comment, marked as answer]
👋 
please update to `>=3.10` 

ref: https://github.com/developmentseed/titiler/pull/374
